symphony-bdk-examples/bdk-app-spring-boot-example/src/main/java/com/symphony/bdk/examples/app/spring/AppSpringApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.symphony.bdk.examples.app.spring;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class AppSpringApplication {
```

Comment:
I'd like to see a real application in action in this module, it could just be a simple `index.html` + `main.js` (under `src/main/resources/static`) in pure Javascript (@symphony-adnane could help 😃) that calls the `AppAuthController` endpoints

Reply:
I would prefer to do it in the next PR 😄

Reply:
+1 that's a must have as we are trying to replace https://github.com/symphonyoss/app-auth-example
(and of course yes it can be done after this PR)

Reply:
You can do it as part of a separate PR once this one merged, but on the same ticket